<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlarmRuleManagementWindow</class>
 <widget class="QMainWindow" name="AlarmRuleManagementWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>告警规则管理</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="ruleTable"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="addRuleButton">
        <property name="text">
         <string>新建规则</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editRuleButton">
        <property name="text">
         <string>编辑规则</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteRuleButton">
        <property name="text">
         <string>删除规则</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui> 